docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Clear the output, we don't want to show warnings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>